datafusion/physical-plan/src/analyze.rs

```diff
@@ -134,9 +134,8 @@ impl ExecutionPlan for AnalyzeExec {
         vec![&self.input]
     }
 
-    /// AnalyzeExec is handled specially so this value is ignored
     fn required_input_distribution(&self) -> Vec<Distribution> {
-        vec![]
+        vec![Distribution::UnspecifiedDistribution]
```

Comment:
This seems rational to me. I'm guessing 1 item per children is expected? Since we know we have 1 child it makes sense to abide by that. If it were a lot more complex or something it'd be justifiable but it's actually less LOC to just implement this right than to leave a comment explaining why it's not.